src/test/java/org/apache/drill_web_test_framework/web_ui/pages/OptionsPage.java

```diff
@@ -0,0 +1,244 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.drill_web_test_framework.web_ui.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class OptionsPage extends BasePage {
+
+  @FindBy(css = "input[id='searchBox']")
+  private WebElement searchField;
+
+  @FindBy(css = "button[title='Clear search']")
+  private WebElement clearSearchButton;
+
+  @FindBy(id = "optionsTbl")
+  private WebElement optionTable;
+
+  @FindBy(xpath = "//button[text()='planner']")
+  private WebElement plannerFilter;
+
+  @FindBy(xpath = "//button[text()='store']")
+  private WebElement storeFilter;
+
+  @FindBy(xpath = "//button[text()='parquet']")
+  private WebElement parquetFilter;
+
+  @FindBy(xpath = "//button[text()='hashagg']")
+  private WebElement hashaggFilter;
+
+  @FindBy(xpath = "//button[text()='hashjoin']")
+  private WebElement hashjoinFilter;
+
+  @FindBy(xpath = "//th[text()='OPTION']")
+  private WebElement optionSortBy;
+
+  @FindBy(xpath = "//th[text()='VALUE']")
+  private WebElement valueSortBy;
+
+  @FindBy(xpath = "//th[text()='DESCRIPTION']")
+  private WebElement descriptionSortBy;
+
+  @FindBy(css = "button[onclick=\"alterSysOptionUsingId('drill.exec.functions.cast_empty_string_to_null')\"]")
```

Comment:
onclick can change eventually and your tests would fail. Instead you can search for the table row with the option, and find the button by column index.

Reply:
Done